### DISCUSSION TITLE
ltsp command failed error when starting the image on the client

### DISCUSSION THREAD

**Discussion by pietus** [ltsp/ltsp#695] [Q&A]
Hi,
sorry alkis for misplacement of this issue. Thanks for the hint. I Hope it's better situated over here.
After a server change to debian bullseye the originally working images don't start anymore with error message:

ltsp command failed: /usr/share/ltsp/client/login/pwmerge -lq /etc/ltsp /etc /etc

I installed ltsp following: [https://ltsp.org/docs/installation/](https://github.com/ltsp/ltsp/issues/url)
and installed virtualbox and its VMs to its locations linked to /srv/ltsp and newly generated the images  with `ltsp image imagename` without problems.

Starting the image from the client ends in upper error.
What am I doing wrong?
pietus

**Comment by alkisg** [comment]
Hi @pietus, sure here is fine.
The pwmerge problem means that your /etc/passwd or /etc/group are malformed, either on your server or in your VM.
Run `sudo pwck` and `sudo grpck` both on your server and in your VM, and fix the issue.

**Comment by pietMS** [comment]
Hi alkis,
I used these commands to solve the problems that indeed existed. Now the reply of the commands are empty. I restarted the server to make sure everything has been excepted, but trying to start a client into an image ends up in the same result. The upper error reveals again.

Anything else I can do?
btw. thank You for Your help!

**Comment by alkisg** [comment]
Did you run `ltsp image`? If not, your published image still contains the errors...

**Comment by pietMS** [comment]
Hi,
renewed all images with `ltsp image [imagename]`, but the error persists It doesn't matter which image I'm trying to load . . .

**Comment by alkisg** [comment]
Upload a screenshot of the error.

**Comment by pietMS** [comment]
Hi,
![IMG_0103](https://user-images.githubusercontent.com/78079960/160477575-a239ef3a-cf5f-4c27-8449-73c54c600761.JPG)
Sorry for the flash in the center - it's too dark to do without it. But the rest looks readable :)

**Comment by alkisg** [comment]
It's still the same issue, your /etc/group is broken.

**Comment by pietMS** [comment]
Hi alkis,
Mhh, I wonder what is going on here. I set back passwd, group and shadow to the point of installation, rebuild the image and the same thing happens . . . What I noticed during this action, bullseye has a gshadow file that is new for me. So I didn't create a backup after installation for that one.

What do You think, what can I do to solve this? Complete new installation, doing all the work all over?

**Comment by alkisg** [comment]
You can set `DEBUG_SHELL=1` in ltsp.conf, and at the point when the problem happens, scp and upload /etc/passwd, /etc/group, /etc/ltsp/passwd,and /etc/ltsp/group. I'll then be able to examine them and tell you which line is wrong.

Or come by the https://ltsp.org/advanced/chat-room/ for live support, while having a VM client handy, where we'll be able to share its screen and reproduce the issue.

**Comment by pietMS** [comment]
Hi alkis,
it looks as if I solved the issue. Following a spontaneous inspiration I deleted everything under /srv/tftp, rebuild all images and run ltsp initrd and ltsp ipxe. Booting this image from a client shows the login screen, finally, without running into the upper error. So on top the problem doesn't s seem to be along the users and their configuration in passwd, group and shadow. There must have been something wrong with tftp-configuration either.
Coming back here, if more of this shows up.
So thanks a lot, alkis,  for Your help, like always :) You're the best

piet
